### DISCUSSION TITLE
How to get past Logic 10.7 plugin validation

### DISCUSSION THREAD

**Discussion by RossoRacer** [DISTRHO/Cardinal#337] [Q&A]
Seems to be a stumbling block for me. Try and rescan and it crashes every time. Any suggestions welcome (relevant).

**Comment by armandvedel** [comment]
I never had any problems but you could try opening logic in rosetta mode and rescanning. this way the intel version of the plugin would be validated first. This technique was used with some early versions of M1 plugins

**Comment by RossoRacer** [comment]
There appears to be no way to open the current version of Logic in Rosetta on my Mac sadly. I looked in the GET INFO panel and that option is not there. My Mac is Intel anyway, not Apple Silicon. But thanks for the response.
